src/client/components/Forms/SignUp.js

```diff
@@ -1,57 +1,171 @@
-  const [email, setEmail] = useState('');
-  const [password, setPassword] = useState('');
-  const [passwordConfirm, setPasswordConfirm] = useState('');
-  const handleEmailInput = (e) => setEmail(e.target.value);
-  const handlePasswordInput = (e) => setPassword(e.target.value);
-  const handlePasswordConfirmInput = (e) => setPasswordConfirm(e.target.value);
-  const handleSubmit = (e) => {
-    e.preventDefault();
-    onSubmit({ email, password, passwordConfirm, fullName: 'Jane Doe' });
-  };
+  const { handleChange, handleSubmit, values, errors } = useForm(
+    validate,
+    onSubmit,
+    {
+      fullName: '',
+      position: '',
+      profileImageUrl: '',
+      linkedin: '',
+      github: '',
+      website: '',
+      email: '',
+      password: '',
+      passwordConfirm: '',
+    },
+  );
+
   return (
-    <form onSubmit={handleSubmit}>
-      <label htmlFor="email">
-        Email:{' '}
+    <form className="form" onSubmit={handleSubmit} noValidate>
+      <h2>Signup to Review Resume</h2>
+      <div className="button-forms">
+        <Button
+          buttonName="Signup With Linkedin"
+          style={{ backgroundColor: '#0676f2' }}
+        />
+        <div className="space">
+          <div className="github-container">
+            <img className="github-logo" src={iconGithub} alt="github icon" />
+          </div>
+        </div>
+      </div>
+      <br />
+      <h3 className="or">Or</h3>
+      <label className="label" htmlFor="fullName">
+        Full name:
+        <input
+          className="input"
+          type="text"
+          name="fullName"
+          value={values.fullName}
+          onChange={handleChange}
+          required
+        />
+      </label>
+      <label className="label" htmlFor="position">
+        Position:
         <input
+          className="input"
+          type="text"
+          name="position"
+          value={values.position}
+          onChange={handleChange}
+          required
+        />
+      </label>
+      <label className="label" htmlFor="email" validate>
+        Email:
+        <input
+          className="input"
           type="email"
           name="email"
           placeholder="Write your email"
-          value={email}
-          onChange={handleEmailInput}
+          value={values.email}
+          onChange={handleChange}
+          required
+        />
+        {errors.email && <p className="validation">{errors.email}</p>}
+      </label>
+      <label className="label" htmlFor="profileImageUrl">
+        Profile picture:
+        <input
+          className="input"
+          type="url"
+          name="profileImageUrl"
+          placeholder="profile image URL"
+          value={values.profileImageUrl}
+          onChange={handleChange}
           required
         />
       </label>
-      <label htmlFor="password">
-        Password:{' '}
+      <label className="label" htmlFor="linkedin">
+        LinkedIn profile:
```

Comment:
```suggestion
        LinkedIn user name:
```

Reply:
Should be a username and therefore, we should not have a type of url. The same applies for the Github field